CHANGELOG.md

```diff
@@ -101,6 +101,8 @@
   [#10983](https://github.com/Kong/kong/pull/10983)
 
 #### PDK
+- Rework pdk lmdb environment information with `kong.node.get_memory_stats`.
```

Comment:
This feature seems to be unreleased, and we should change the existing changelog instead.